actors/runtime/runtime.go

```diff
@@ -14,6 +14,22 @@ import (
 	exitcode "github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
 )
 
+type LogLevel int
+
+const (
+	// DebugLevel logs are typically voluminous, and are usually disabled in
+	// production.
+	DEBUG = iota
```

Comment:
Could you please confirm that these integer values match those for <relevant logging framework in use by Lotus>, and document that they do here?

Reply:
Assuming this meant levels should match uber-go/zap (which is where Lotus gets it's log levels). Done.